plugins/astroturf/README.md

```diff
@@ -0,0 +1,50 @@
+# @poi/plugin-astroturf
+
+Use [astroturf](https://github.com/4Catalyzer/astroturf) to write CSS in JS with zero-runtime.
+
+## Install
+
+```bash
+yarn add astroturf @poi/plugin-astroturf --dev
```

Comment:
@egoist Actually, astroturf still have some "runtime" when using it's [Component API](https://github.com/4Catalyzer/astroturf#component-api)

```jsx
import styled, { css } from 'astroturf';

const Button = styled('button')`
  color: black;
  border: 1px solid black;
  background-color: white;

  &.primary {
    color: blue;
    border: 1px solid blue;
  }

  &.color-green {
    color: green;
  }
`;
```

The [`styled`](https://github.com/4Catalyzer/astroturf/blob/5664f9c080db645f0579153cd7e96522fe5515b4/src/runtime/styled.js#L12) function will create a React component at runtime.